x-pack/test/api_integration/apis/security/roles.ts

```diff
@@ -391,6 +392,87 @@ export default function ({ getService }: FtrProviderContext) {
             _unrecognized_applications: ['logstash-default'],
           });
       });
+
+      it('should get roles by space id', async () => {
```

Comment:
Nit: add test for no space ID match/empty return

Reply:
There is a unit test for it already https://github.com/elastic/kibana/pull/181165/files#diff-c5613ca58368af4bcb37814971d4a09aa66a4265a1ed8c07b96f13f5da4b5b44R139